.travis.yml

```diff
@@ -36,18 +37,24 @@ install:
 before_script:
   - cp src/test/resources/config/keystore.jks src/main/resources/config/keystore.jks # we need the keystore to run our app, and we need our app to run e2e tests
   - chmod +x src/test/bash/*.sh
+  - chmod +x util/*.sh
   - ./gradlew bootRun 1> /dev/null &
   - sleep 60 # give spring boot some time to start
 script:
   - ./gradlew test
   - yarn test
   - yarn e2e
   - ./gradlew bootRepackage -Pprod buildDocker -x test
+  - ./gradlew generateOpenApiSpec
+  - ./gradlew generateJavaClient
+  - echo "include 'managementportal-client'" >> settings.gradle # make this a sub-project so we can build artifacts and javadoc easily
   - killall java # stop running MP instance
   - ./src/test/bash/run-prod-e2e.sh
   - ./src/test/bash/test-docker-build-on-release-branch.sh
 after_script:
   - ./gradlew sendCoverageToCodacy
+after_success:
+  - ./util/update-gh-pages-docs.sh
```

Comment:
you can run this as a deploy target instead, and limit it to the master branch (or even just tags)

Reply:
This could be removed now?

Reply:
Yes. My bad. Pushed the update